OCG, Project Plan edits, etc. (#14)
Capstone-Skynet/Capstone-Skynet.github.io: docs/proposal/2_1_ocg_objectives.tex

The main objectives are sorted by the client's priority in descending order.

\textbf{Integration of Electronics with the Drone}:
Since the ultimate deliverable consists of a highly mobile computing platform,

> our uttermost most important objective is a successful integration of computation and processing hardware with the drone.

Reviewer: ```suggestion
our utmost objective is a successful integration of computation and processing hardware with the drone.
```

> The equipment for capturing vidoe data, perform processing, and trasmitting digital data should be reasonbly compact and easily deployable.

Reviewer: ```suggestion
The equipment for capturing video data, performing processing, and transmitting digital data should be reasonably compact and easily deployable.
```

Author: Lmao 3x in one sentence. Guess I shorted

> the total power draw from the drone as well as power output, in terms of thermal output, should be reasonable such that they affect minimally

Reviewer: ```suggestion
The thermal characteristics and total power consumption of the device should be reasonable such that they affect minimally
```

on the flight characteristics of the drone.

\textbf{Air-to-Ground Data Transmission}:
Critical to research and analytics, we need to ensure the drone payload is able to transmit video sensor data, as well as the processed data

> from the onboard hardware accelerator (FPGA). There is a ground station, consist of a receiver and a display device such as a mobile phone or 

Reviewer: ```suggestion
from the onboard hardware accelerator (FPGA). There is a ground station, consisting of a receiver and a display device (such as a mobile phone or 
```

> computer receives and decode the transmission for display, providing the client or user with real-time video stream and machine learning intents.

Reviewer: ```suggestion
computer) receives and decodes the transmission for display, providing the user with a real-time video stream overlaid with machine learning intents.
```

> The wireless frequency and band chosen is compliant with local laws; no further actions are required if we utilize WiFi protocol (802.11n/ac in 2.4GHz or 5.2GHz)

Reviewer: ```suggestion
The wireless frequency and band chosen is compliant with local laws; no further actions are required if we utilize the WiFi protocol (802.11n/ac in 2.4GHz or 5.2GHz)
```

using off-the-shelf WiFi modules.

\textbf{Machine Learning Implementation on the FPGA}:

> The hardware-accelerated computing platform interfaces with attached peripherals such as the camera sensor, the data transmitter (DTX), and other hardware properly

Reviewer: ```suggestion
The hardware-accelerated computing platform correctly interfaces with attached peripherals such as the camera sensor, the data transmitter (DTX), and other necessary hardware
```

> using well-defined communication protocol, such as serial bus or PCI-E, whichever one that is suitable for the data bandwidth requirements. 

Reviewer: ```suggestion
through use of well-defined communication protocols, such as serial or PCI-E (whichever is suitable given bandwidth requirements). 
```

> The FPGA should feature a starter ML model, such as but not limited to CNN, RNN, or YOLO, fully capable of basic computer vision.

Reviewer: ```suggestion
The FPGA should feature a starter ML model, such as (but not limited to) CNN, RNN, or YOLO, fully capable of basic computer vision.
```

> The netlist design for the ML models are adequately synthesized with efficient datapath.

Reviewer: ```suggestion
The netlist design for the ML models are adequately synthesized, resulting in an efficient datapath.
```
